server/backend/quiz/migrations/0002_initial.py

```
@@ -0,0 +1,50 @@
+
+    dependencies = [
+        ("quiz", "0001_initial"),
+        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="quizattempt",
+            name="user",
+            field=models.ForeignKey(
+                on_delete=django.db.models.deletion.CASCADE, to=settings.AUTH_USER_MODEL
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)